quickstart/ONECLICK-INSTRUCTION-BOX.md

```diff
@@ -0,0 +1,11 @@
+### Instructions

```

Comment:
This file's name should be lowercase only, as OneClick is case sensitive on the matter